<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />

<meta name="viewport" content="width=device-width, initial-scale=1" />


<meta name="date" content="2021-06-02" />

<title>Data sets for download / own use</title>

<script src="data:application/javascript;base64,Ly8gUGFuZG9jIDIuOSBhZGRzIGF0dHJpYnV0ZXMgb24gYm90aCBoZWFkZXIgYW5kIGRpdi4gV2UgcmVtb3ZlIHRoZSBmb3JtZXIgKHRvCi8vIGJlIGNvbXBhdGlibGUgd2l0aCB0aGUgYmVoYXZpb3Igb2YgUGFuZG9jIDwgMi44KS4KZG9jdW1lbnQuYWRkRXZlbnRMaXN0ZW5lcignRE9NQ29udGVudExvYWRlZCcsIGZ1bmN0aW9uKGUpIHsKICB2YXIgaHMgPSBkb2N1bWVudC5xdWVyeVNlbGVjdG9yQWxsKCJkaXYuc2VjdGlvbltjbGFzcyo9J2xldmVsJ10gPiA6Zmlyc3QtY2hpbGQiKTsKICB2YXIgaSwgaCwgYTsKICBmb3IgKGkgPSAwOyBpIDwgaHMubGVuZ3RoOyBpKyspIHsKICAgIGggPSBoc1tpXTsKICAgIGlmICghL15oWzEtNl0kL2kudGVzdChoLnRhZ05hbWUpKSBjb250aW51ZTsgIC8vIGl0IHNob3VsZCBiZSBhIGhlYWRlciBoMS1oNgogICAgYSA9IGguYXR0cmlidXRlczsKICAgIHdoaWxlIChhLmxlbmd0aCA+IDApIGgucmVtb3ZlQXR0cmlidXRlKGFbMF0ubmFtZSk7CiAgfQp9KTsK"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>






<link rel="stylesheet" href="data:text/css,body%20%7B%0Abackground%2Dcolor%3A%20%23fff%3B%0Amargin%3A%201em%20auto%3B%0Amax%2Dwidth%3A%20700px%3B%0Aoverflow%3A%20visible%3B%0Apadding%2Dleft%3A%202em%3B%0Apadding%2Dright%3A%202em%3B%0Afont%2Dfamily%3A%20%22Open%20Sans%22%2C%20%22Helvetica%20Neue%22%2C%20Helvetica%2C%20Arial%2C%20sans%2Dserif%3B%0Afont%2Dsize%3A%2014px%3B%0Aline%2Dheight%3A%201%2E35%3B%0A%7D%0A%23TOC%20%7B%0Aclear%3A%20both%3B%0Amargin%3A%200%200%2010px%2010px%3B%0Apadding%3A%204px%3B%0Awidth%3A%20400px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Aborder%2Dradius%3A%205px%3B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Afont%2Dsize%3A%2013px%3B%0Aline%2Dheight%3A%201%2E3%3B%0A%7D%0A%23TOC%20%2Etoctitle%20%7B%0Afont%2Dweight%3A%20bold%3B%0Afont%2Dsize%3A%2015px%3B%0Amargin%2Dleft%3A%205px%3B%0A%7D%0A%23TOC%20ul%20%7B%0Apadding%2Dleft%3A%2040px%3B%0Amargin%2Dleft%3A%20%2D1%2E5em%3B%0Amargin%2Dtop%3A%205px%3B%0Amargin%2Dbottom%3A%205px%3B%0A%7D%0A%23TOC%20ul%20ul%20%7B%0Amargin%2Dleft%3A%20%2D2em%3B%0A%7D%0A%23TOC%20li%20%7B%0Aline%2Dheight%3A%2016px%3B%0A%7D%0Atable%20%7B%0Amargin%3A%201em%20auto%3B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dcolor%3A%20%23DDDDDD%3B%0Aborder%2Dstyle%3A%20outset%3B%0Aborder%2Dcollapse%3A%20collapse%3B%0A%7D%0Atable%20th%20%7B%0Aborder%2Dwidth%3A%202px%3B%0Apadding%3A%205px%3B%0Aborder%2Dstyle%3A%20inset%3B%0A%7D%0Atable%20td%20%7B%0Aborder%2Dwidth%3A%201px%3B%0Aborder%2Dstyle%3A%20inset%3B%0Aline%2Dheight%3A%2018px%3B%0Apadding%3A%205px%205px%3B%0A%7D%0Atable%2C%20table%20th%2C%20table%20td%20%7B%0Aborder%2Dleft%2Dstyle%3A%20none%3B%0Aborder%2Dright%2Dstyle%3A%20none%3B%0A%7D%0Atable%20thead%2C%20table%20tr%2Eeven%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Ap%20%7B%0Amargin%3A%200%2E5em%200%3B%0A%7D%0Ablockquote%20%7B%0Abackground%2Dcolor%3A%20%23f6f6f6%3B%0Apadding%3A%200%2E25em%200%2E75em%3B%0A%7D%0Ahr%20%7B%0Aborder%2Dstyle%3A%20solid%3B%0Aborder%3A%20none%3B%0Aborder%2Dtop%3A%201px%20solid%20%23777%3B%0Amargin%3A%2028px%200%3B%0A%7D%0Adl%20%7B%0Amargin%2Dleft%3A%200%3B%0A%7D%0Adl%20dd%20%7B%0Amargin%2Dbottom%3A%2013px%3B%0Amargin%2Dleft%3A%2013px%3B%0A%7D%0Adl%20dt%20%7B%0Afont%2Dweight%3A%20bold%3B%0A%7D%0Aul%20%7B%0Amargin%2Dtop%3A%200%3B%0A%7D%0Aul%20li%20%7B%0Alist%2Dstyle%3A%20circle%20outside%3B%0A%7D%0Aul%20ul%20%7B%0Amargin%2Dbottom%3A%200%3B%0A%7D%0Apre%2C%20code%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0Aborder%2Dradius%3A%203px%3B%0Acolor%3A%20%23333%3B%0Awhite%2Dspace%3A%20pre%2Dwrap%3B%20%0A%7D%0Apre%20%7B%0Aborder%2Dradius%3A%203px%3B%0Amargin%3A%205px%200px%2010px%200px%3B%0Apadding%3A%2010px%3B%0A%7D%0Apre%3Anot%28%5Bclass%5D%29%20%7B%0Abackground%2Dcolor%3A%20%23f7f7f7%3B%0A%7D%0Acode%20%7B%0Afont%2Dfamily%3A%20Consolas%2C%20Monaco%2C%20%27Courier%20New%27%2C%20monospace%3B%0Afont%2Dsize%3A%2085%25%3B%0A%7D%0Ap%20%3E%20code%2C%20li%20%3E%20code%20%7B%0Apadding%3A%202px%200px%3B%0A%7D%0Adiv%2Efigure%20%7B%0Atext%2Dalign%3A%20center%3B%0A%7D%0Aimg%20%7B%0Abackground%2Dcolor%3A%20%23FFFFFF%3B%0Apadding%3A%202px%3B%0Aborder%3A%201px%20solid%20%23DDDDDD%3B%0Aborder%2Dradius%3A%203px%3B%0Aborder%3A%201px%20solid%20%23CCCCCC%3B%0Amargin%3A%200%205px%3B%0A%7D%0Ah1%20%7B%0Amargin%2Dtop%3A%200%3B%0Afont%2Dsize%3A%2035px%3B%0Aline%2Dheight%3A%2040px%3B%0A%7D%0Ah2%20%7B%0Aborder%2Dbottom%3A%204px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Apadding%2Dbottom%3A%202px%3B%0Afont%2Dsize%3A%20145%25%3B%0A%7D%0Ah3%20%7B%0Aborder%2Dbottom%3A%202px%20solid%20%23f7f7f7%3B%0Apadding%2Dtop%3A%2010px%3B%0Afont%2Dsize%3A%20120%25%3B%0A%7D%0Ah4%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23f7f7f7%3B%0Amargin%2Dleft%3A%208px%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Ah5%2C%20h6%20%7B%0Aborder%2Dbottom%3A%201px%20solid%20%23ccc%3B%0Afont%2Dsize%3A%20105%25%3B%0A%7D%0Aa%20%7B%0Acolor%3A%20%230033dd%3B%0Atext%2Ddecoration%3A%20none%3B%0A%7D%0Aa%3Ahover%20%7B%0Acolor%3A%20%236666ff%3B%20%7D%0Aa%3Avisited%20%7B%0Acolor%3A%20%23800080%3B%20%7D%0Aa%3Avisited%3Ahover%20%7B%0Acolor%3A%20%23BB00BB%3B%20%7D%0Aa%5Bhref%5E%3D%22http%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0Aa%5Bhref%5E%3D%22https%3A%22%5D%20%7B%0Atext%2Ddecoration%3A%20underline%3B%20%7D%0A%0Acode%20%3E%20span%2Ekw%20%7B%20color%3A%20%23555%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Edt%20%7B%20color%3A%20%23902000%3B%20%7D%20%0Acode%20%3E%20span%2Edv%20%7B%20color%3A%20%2340a070%3B%20%7D%20%0Acode%20%3E%20span%2Ebn%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Efl%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Ech%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Est%20%7B%20color%3A%20%23d14%3B%20%7D%20%0Acode%20%3E%20span%2Eco%20%7B%20color%3A%20%23888888%3B%20font%2Dstyle%3A%20italic%3B%20%7D%20%0Acode%20%3E%20span%2Eot%20%7B%20color%3A%20%23007020%3B%20%7D%20%0Acode%20%3E%20span%2Eal%20%7B%20color%3A%20%23ff0000%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Efu%20%7B%20color%3A%20%23900%3B%20font%2Dweight%3A%20bold%3B%20%7D%20%0Acode%20%3E%20span%2Eer%20%7B%20color%3A%20%23a61717%3B%20background%2Dcolor%3A%20%23e3d2d2%3B%20%7D%20%0A" type="text/css" />




</head>

<body>




<h1 class="title toc-ignore">Data sets for download / own use</h1>
<h4 class="date">02 June 2021</h4>



<p>All reference data (about microorganisms, antibiotics, R/SI interpretation, EUCAST rules, etc.) in this <code>AMR</code> package are reliable, up-to-date and freely available. We continually export our data sets to formats for use in R, SPSS, SAS, Stata and Excel. We also supply tab separated files that are machine-readable and suitable for input in any software program, such as laboratory information systems.</p>
<p>On this page, we explain how to download them and how the structure of the data sets look like.</p>
<p class="dataset-within-r">
If you are reading this page from within R, please <a href="https://msberends.github.io/AMR/articles/datasets.html">visit our website</a>, which is automatically updated with every code change.
</p>
<div id="microorganisms-currently-accepted-names" class="section level2">
<h2>Microorganisms (currently accepted names)</h2>
<p>A data set with 70,026 rows and 16 columns, containing the following column names:<br />
<em>mo</em>, <em>fullname</em>, <em>kingdom</em>, <em>phylum</em>, <em>class</em>, <em>order</em>, <em>family</em>, <em>genus</em>, <em>species</em>, <em>subspecies</em>, <em>rank</em>, <em>ref</em>, <em>species_id</em>, <em>source</em>, <em>prevalence</em> and <em>snomed</em>.</p>
<p>This data set is in R available as <code>microorganisms</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 11 March 2021 20:59:32 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/microorganisms.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.rds">R file</a> (2.2 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.xlsx">Excel file</a> (6.4 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.txt">plain text file</a> (14.8 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.sas">SAS file</a> (27.4 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.sav">SPSS file</a> (27.8 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.dta">Stata file</a> (25.1 MB)</li>
</ul>
<p><strong>NOTE: The exported files for SAS, SPSS and Stata do not contain SNOMED codes, as their file size would exceed 100 MB; the file size limit of GitHub.</strong> Advice? Use R instead.</p>
<div id="source" class="section level3">
<h3>Source</h3>
<p>Our full taxonomy of microorganisms is based on the authoritative and comprehensive:</p>
<ul>
<li><a href="http://www.catalogueoflife.org">Catalogue of Life</a> (included version: 2019)</li>
<li><a href="https://lpsn.dsmz.de">List of Prokaryotic names with Standing in Nomenclature</a> (LPSN, last updated: March 2021)</li>
<li>US Edition of SNOMED CT from 1 September 2020, retrieved from the <a href="https://phinvads.cdc.gov/vads/ViewValueSet.action?oid=2.16.840.1.114222.4.11.1009">Public Health Information Network Vocabulary Access and Distribution System (PHIN VADS)</a>, OID 2.16.840.1.114222.4.11.1009, version 12</li>
</ul>
</div>
<div id="example-content" class="section level3">
<h3>Example content</h3>
<p>Included (sub)species per taxonomic kingdom:</p>
<table>
<thead>
<tr class="header">
<th align="center">Kingdom</th>
<th align="center">Number of (sub)species</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">(unknown kingdom)</td>
<td align="center">1</td>
</tr>
<tr class="even">
<td align="center">Animalia</td>
<td align="center">2,153</td>
</tr>
<tr class="odd">
<td align="center">Archaea</td>
<td align="center">694</td>
</tr>
<tr class="even">
<td align="center">Bacteria</td>
<td align="center">22,118</td>
</tr>
<tr class="odd">
<td align="center">Chromista</td>
<td align="center">32,167</td>
</tr>
<tr class="even">
<td align="center">Fungi</td>
<td align="center">9,582</td>
</tr>
</tbody>
</table>
<p>Example rows when filtering on genus <em>Escherichia</em>:</p>
<table>
<colgroup>
<col width="4%" />
<col width="8%" />
<col width="3%" />
<col width="5%" />
<col width="7%" />
<col width="6%" />
<col width="6%" />
<col width="4%" />
<col width="4%" />
<col width="4%" />
<col width="3%" />
<col width="8%" />
<col width="11%" />
<col width="2%" />
<col width="4%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">mo</th>
<th align="center">fullname</th>
<th align="center">kingdom</th>
<th align="center">phylum</th>
<th align="center">class</th>
<th align="center">order</th>
<th align="center">family</th>
<th align="center">genus</th>
<th align="center">species</th>
<th align="center">subspecies</th>
<th align="center">rank</th>
<th align="center">ref</th>
<th align="center">species_id</th>
<th align="center">source</th>
<th align="center">prevalence</th>
<th align="center">snomed</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">B_ESCHR</td>
<td align="center">Escherichia</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">genus</td>
<td align="center"></td>
<td align="center">bc4fdde6867d5ecfc728000b0bfb49a3</td>
<td align="center">CoL</td>
<td align="center">1</td>
<td align="center">64735005</td>
</tr>
<tr class="even">
<td align="center">B_ESCHR_ALBR</td>
<td align="center">Escherichia albertii</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center">albertii</td>
<td align="center"></td>
<td align="center">species</td>
<td align="center">Huys et al., 2003</td>
<td align="center">36618b1ed3b8b7e5a61f40eb9386e63c</td>
<td align="center">CoL</td>
<td align="center">1</td>
<td align="center">419388003</td>
</tr>
<tr class="odd">
<td align="center">B_ESCHR_COLI</td>
<td align="center">Escherichia coli</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center">coli</td>
<td align="center"></td>
<td align="center">species</td>
<td align="center">Castellani et al., 1919</td>
<td align="center">3254b3db31bf16fdde669ac57bf8c4fe</td>
<td align="center">CoL</td>
<td align="center">1</td>
<td align="center">1095001000112106, 715307006, 737528008, …</td>
</tr>
<tr class="even">
<td align="center">B_ESCHR_FRGS</td>
<td align="center">Escherichia fergusonii</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center">fergusonii</td>
<td align="center"></td>
<td align="center">species</td>
<td align="center">Farmer et al., 1985</td>
<td align="center">82d98b10c456ce5f4c8c515f4e1567e2</td>
<td align="center">CoL</td>
<td align="center">1</td>
<td align="center">72461005</td>
</tr>
<tr class="odd">
<td align="center">B_ESCHR_HRMN</td>
<td align="center">Escherichia hermannii</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center">hermannii</td>
<td align="center"></td>
<td align="center">species</td>
<td align="center">Brenner et al., 1983</td>
<td align="center">b16086aee36e3b46b565510083ab4b65</td>
<td align="center">CoL</td>
<td align="center">1</td>
<td align="center">85786000</td>
</tr>
<tr class="even">
<td align="center">B_ESCHR_MRMT</td>
<td align="center">Escherichia marmotae</td>
<td align="center">Bacteria</td>
<td align="center">Proteobacteria</td>
<td align="center">Gammaproteobacteria</td>
<td align="center">Enterobacterales</td>
<td align="center">Enterobacteriaceae</td>
<td align="center">Escherichia</td>
<td align="center">marmotae</td>
<td align="center"></td>
<td align="center">species</td>
<td align="center">Liu et al., 2015</td>
<td align="center">792928</td>
<td align="center">LPSN</td>
<td align="center">1</td>
<td align="center">14961000146107</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="microorganisms-previously-accepted-names" class="section level2">
<h2>Microorganisms (previously accepted names)</h2>
<p>A data set with 14,100 rows and 4 columns, containing the following column names:<br />
<em>fullname</em>, <em>fullname_new</em>, <em>ref</em> and <em>prevalence</em>.</p>
<p><strong>Note:</strong> remember that the ‘ref’ columns contains the scientific reference to the old taxonomic entries, i.e. of column <em>‘fullname’</em>. For the scientific reference of the new names, i.e. of column <em>‘fullname_new’</em>, see the <code>microorganisms</code> data set.</p>
<p>This data set is in R available as <code>microorganisms.old</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 5 March 2021 09:46:55 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/microorganisms.old.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.rds">R file</a> (0.2 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.xlsx">Excel file</a> (0.5 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.txt">plain text file</a> (0.9 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.sas">SAS file</a> (2.1 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.sav">SPSS file</a> (2.2 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/microorganisms.old.dta">Stata file</a> (2 MB)</li>
</ul>
<div id="source-1" class="section level3">
<h3>Source</h3>
<p>This data set contains old, previously accepted taxonomic names. The data sources are the same as the <code>microorganisms</code> data set:</p>
<ul>
<li><a href="http://www.catalogueoflife.org">Catalogue of Life</a> (included version: 2019)</li>
<li><a href="https://lpsn.dsmz.de">List of Prokaryotic names with Standing in Nomenclature</a> (LPSN, last updated: March 2021)</li>
</ul>
</div>
<div id="example-content-1" class="section level3">
<h3>Example content</h3>
<p>Example rows when filtering on <em>Escherichia</em>:</p>
<table>
<thead>
<tr class="header">
<th align="center">fullname</th>
<th align="center">fullname_new</th>
<th align="center">ref</th>
<th align="center">prevalence</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Escherichia adecarboxylata</td>
<td align="center">Leclercia adecarboxylata</td>
<td align="center">Leclerc, 1962</td>
<td align="center">1</td>
</tr>
<tr class="even">
<td align="center">Escherichia blattae</td>
<td align="center">Shimwellia blattae</td>
<td align="center">Burgess et al., 1973</td>
<td align="center">1</td>
</tr>
<tr class="odd">
<td align="center">Escherichia vulneris</td>
<td align="center">Pseudescherichia vulneris</td>
<td align="center">Brenner et al., 1983</td>
<td align="center">1</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="antibiotic-agents" class="section level2">
<h2>Antibiotic agents</h2>
<p>A data set with 456 rows and 14 columns, containing the following column names:<br />
<em>ab</em>, <em>atc</em>, <em>cid</em>, <em>name</em>, <em>group</em>, <em>atc_group1</em>, <em>atc_group2</em>, <em>abbreviations</em>, <em>synonyms</em>, <em>oral_ddd</em>, <em>oral_units</em>, <em>iv_ddd</em>, <em>iv_units</em> and <em>loinc</em>.</p>
<p>This data set is in R available as <code>antibiotics</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 4 May 2021 13:38:27 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/antibiotics.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.rds">R file</a> (32 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.xlsx">Excel file</a> (65 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.txt">plain text file</a> (0.1 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.sas">SAS file</a> (1.8 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.sav">SPSS file</a> (1.3 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antibiotics.dta">Stata file</a> (0.3 MB)</li>
</ul>
<div id="source-2" class="section level3">
<h3>Source</h3>
<p>This data set contains all EARS-Net and ATC codes gathered from WHO and WHONET, and all compound IDs from PubChem. It also contains all brand names (synonyms) as found on PubChem and Defined Daily Doses (DDDs) for oral and parenteral administration.</p>
<ul>
<li><a href="https://www.whocc.no/atc_ddd_index/">ATC/DDD index from WHO Collaborating Centre for Drug Statistics Methodology</a> (note: this may not be used for commercial purposes, but is freely available from the WHO CC website for personal use)</li>
<li><a href="https://pubchem.ncbi.nlm.nih.gov">PubChem by the US National Library of Medicine</a></li>
<li><a href="https://whonet.org">WHONET software 2019</a></li>
</ul>
</div>
<div id="example-content-2" class="section level3">
<h3>Example content</h3>
<table style="width:100%;">
<colgroup>
<col width="1%" />
<col width="2%" />
<col width="3%" />
<col width="8%" />
<col width="7%" />
<col width="13%" />
<col width="18%" />
<col width="6%" />
<col width="14%" />
<col width="3%" />
<col width="3%" />
<col width="2%" />
<col width="3%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">ab</th>
<th align="center">atc</th>
<th align="center">cid</th>
<th align="center">name</th>
<th align="center">group</th>
<th align="center">atc_group1</th>
<th align="center">atc_group2</th>
<th align="center">abbreviations</th>
<th align="center">synonyms</th>
<th align="center">oral_ddd</th>
<th align="center">oral_units</th>
<th align="center">iv_ddd</th>
<th align="center">iv_units</th>
<th align="center">loinc</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">AMK</td>
<td align="center">J01GB06</td>
<td align="center">37768</td>
<td align="center">Amikacin</td>
<td align="center">Aminoglycosides</td>
<td align="center">Aminoglycoside antibacterials</td>
<td align="center">Other aminoglycosides</td>
<td align="center">ak, ami, amik, …</td>
<td align="center">amicacin, amikacillin, amikacin, …</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">1.0</td>
<td align="center">g</td>
<td align="center">13546-7, 15098-7, 17798-0, …</td>
</tr>
<tr class="even">
<td align="center">AMX</td>
<td align="center">J01CA04</td>
<td align="center">33613</td>
<td align="center">Amoxicillin</td>
<td align="center">Beta-lactams/penicillins</td>
<td align="center">Beta-lactam antibacterials, penicillins</td>
<td align="center">Penicillins with extended spectrum</td>
<td align="center">ac, amox, amx</td>
<td align="center">actimoxi, amoclen, amolin, …</td>
<td align="center">1.5</td>
<td align="center">g</td>
<td align="center">3.0</td>
<td align="center">g</td>
<td align="center">16365-9, 25274-2, 3344-9, …</td>
</tr>
<tr class="odd">
<td align="center">AMC</td>
<td align="center">J01CR02</td>
<td align="center">23665637</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Beta-lactams/penicillins</td>
<td align="center">Beta-lactam antibacterials, penicillins</td>
<td align="center">Combinations of penicillins, incl. beta-lactamase inhibitors</td>
<td align="center">a/c, amcl, aml, …</td>
<td align="center">amocla, amoclan, amoclav, …</td>
<td align="center">1.5</td>
<td align="center">g</td>
<td align="center">3.0</td>
<td align="center">g</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">AMP</td>
<td align="center">J01CA01</td>
<td align="center">6249</td>
<td align="center">Ampicillin</td>
<td align="center">Beta-lactams/penicillins</td>
<td align="center">Beta-lactam antibacterials, penicillins</td>
<td align="center">Penicillins with extended spectrum</td>
<td align="center">am, amp, ampi</td>
<td align="center">acillin, adobacillin, amblosin, …</td>
<td align="center">2.0</td>
<td align="center">g</td>
<td align="center">6.0</td>
<td align="center">g</td>
<td align="center">21066-6, 3355-5, 33562-0, …</td>
</tr>
<tr class="odd">
<td align="center">AZM</td>
<td align="center">J01FA10</td>
<td align="center">447043</td>
<td align="center">Azithromycin</td>
<td align="center">Macrolides/lincosamides</td>
<td align="center">Macrolides, lincosamides and streptogramins</td>
<td align="center">Macrolides</td>
<td align="center">az, azi, azit, …</td>
<td align="center">aritromicina, azasite, azenil, …</td>
<td align="center">0.3</td>
<td align="center">g</td>
<td align="center">0.5</td>
<td align="center">g</td>
<td align="center">16420-2, 25233-8</td>
</tr>
<tr class="even">
<td align="center">PEN</td>
<td align="center">J01CE01</td>
<td align="center">5904</td>
<td align="center">Benzylpenicillin</td>
<td align="center">Beta-lactams/penicillins</td>
<td align="center">Combinations of antibacterials</td>
<td align="center">Combinations of antibacterials</td>
<td align="center">bepe, pen, peni, …</td>
<td align="center">abbocillin, ayercillin, bencilpenicilina, …</td>
<td align="center"></td>
<td align="center"></td>
<td align="center">3.6</td>
<td align="center">g</td>
<td align="center">3913-1</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="antiviral-agents" class="section level2">
<h2>Antiviral agents</h2>
<p>A data set with 102 rows and 9 columns, containing the following column names:<br />
<em>atc</em>, <em>cid</em>, <em>name</em>, <em>atc_group</em>, <em>synonyms</em>, <em>oral_ddd</em>, <em>oral_units</em>, <em>iv_ddd</em> and <em>iv_units</em>.</p>
<p>This data set is in R available as <code>antivirals</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 29 August 2020 19:53:07 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/antibiotics.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.rds">R file</a> (5 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.xlsx">Excel file</a> (14 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.txt">plain text file</a> (16 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.sas">SAS file</a> (80 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.sav">SPSS file</a> (68 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/antivirals.dta">Stata file</a> (67 kB)</li>
</ul>
<div id="source-3" class="section level3">
<h3>Source</h3>
<p>This data set contains all ATC codes gathered from WHO and all compound IDs from PubChem. It also contains all brand names (synonyms) as found on PubChem and Defined Daily Doses (DDDs) for oral and parenteral administration.</p>
<ul>
<li><a href="https://www.whocc.no/atc_ddd_index/">ATC/DDD index from WHO Collaborating Centre for Drug Statistics Methodology</a> (note: this may not be used for commercial purposes, but is freely available from the WHO CC website for personal use)</li>
<li><a href="https://pubchem.ncbi.nlm.nih.gov">PubChem by the US National Library of Medicine</a></li>
</ul>
</div>
<div id="example-content-3" class="section level3">
<h3>Example content</h3>
<table>
<colgroup>
<col width="4%" />
<col width="5%" />
<col width="9%" />
<col width="32%" />
<col width="30%" />
<col width="4%" />
<col width="5%" />
<col width="3%" />
<col width="4%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">atc</th>
<th align="center">cid</th>
<th align="center">name</th>
<th align="center">atc_group</th>
<th align="center">synonyms</th>
<th align="center">oral_ddd</th>
<th align="center">oral_units</th>
<th align="center">iv_ddd</th>
<th align="center">iv_units</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">J05AF06</td>
<td align="center">441300</td>
<td align="center">Abacavir</td>
<td align="center">Nucleoside and nucleotide reverse transcriptase inhibitors</td>
<td align="center">Abacavir, Abacavir sulfate, Ziagen</td>
<td align="center">0.6</td>
<td align="center">g</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">J05AB01</td>
<td align="center">135398513</td>
<td align="center">Aciclovir</td>
<td align="center">Nucleosides and nucleotides excl. reverse transcriptase inhibitors</td>
<td align="center">Acicloftal, Aciclovier, Aciclovir, …</td>
<td align="center">4.0</td>
<td align="center">g</td>
<td align="center">4</td>
<td align="center">g</td>
</tr>
<tr class="odd">
<td align="center">J05AF08</td>
<td align="center">60871</td>
<td align="center">Adefovir dipivoxil</td>
<td align="center">Nucleoside and nucleotide reverse transcriptase inhibitors</td>
<td align="center">Adefovir di ester, Adefovir dipivoxil, Adefovir Dipivoxil, …</td>
<td align="center">10.0</td>
<td align="center">mg</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">J05AE05</td>
<td align="center">65016</td>
<td align="center">Amprenavir</td>
<td align="center">Protease inhibitors</td>
<td align="center">Agenerase, Amprenavir, Amprenavirum, …</td>
<td align="center">1.2</td>
<td align="center">g</td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="center">J05AP06</td>
<td align="center">16076883</td>
<td align="center">Asunaprevir</td>
<td align="center">Antivirals for treatment of HCV infections</td>
<td align="center">Asunaprevir, Sunvepra</td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="center">J05AE08</td>
<td align="center">148192</td>
<td align="center">Atazanavir</td>
<td align="center">Protease inhibitors</td>
<td align="center">Atazanavir, Atazanavir Base, Latazanavir, …</td>
<td align="center">0.3</td>
<td align="center">g</td>
<td align="center"></td>
<td align="center"></td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="intrinsic-bacterial-resistance" class="section level2">
<h2>Intrinsic bacterial resistance</h2>
<p>A data set with 93,892 rows and 2 columns, containing the following column names:<br />
<em>microorganism</em> and <em>antibiotic</em>.</p>
<p>This data set is in R available as <code>intrinsic_resistant</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 5 March 2021 09:46:55 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/intrinsic_resistant.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.rds">R file</a> (69 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.xlsx">Excel file</a> (0.9 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.txt">plain text file</a> (3.5 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.sas">SAS file</a> (7.1 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.sav">SPSS file</a> (7.9 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/intrinsic_resistant.dta">Stata file</a> (7 MB)</li>
</ul>
<div id="source-4" class="section level3">
<h3>Source</h3>
<p>This data set contains all defined intrinsic resistance by EUCAST of all bug-drug combinations, and is based on <a href="https://www.eucast.org/expert_rules_and_intrinsic_resistance/">‘EUCAST Expert Rules’ and ‘EUCAST Intrinsic Resistance and Unusual Phenotypes’ v3.2</a> (2020).</p>
</div>
<div id="example-content-4" class="section level3">
<h3>Example content</h3>
<p>Example rows when filtering on <em>Enterobacter cloacae</em>:</p>
<table>
<thead>
<tr class="header">
<th align="center">microorganism</th>
<th align="center">antibiotic</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Amoxicillin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Amoxicillin/clavulanic acid</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Ampicillin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Ampicillin/sulbactam</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Avoparcin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Azithromycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cefadroxil</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cefazolin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cefoxitin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cephalexin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cephalothin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Clarithromycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Clindamycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Cycloserine</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Dalbavancin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Dirithromycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Erythromycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Flurithromycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Fusidic acid</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Josamycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Lincomycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Linezolid</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Midecamycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Miocamycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Norvancomycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Oleandomycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Oritavancin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Benzylpenicillin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Pirlimycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Pristinamycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Quinupristin/dalfopristin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Ramoplanin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Rifampicin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Rokitamycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Roxithromycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Spiramycin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Tedizolid</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Teicoplanin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Teicoplanin-macromethod</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Telavancin</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Telithromycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Thiacetazone</td>
</tr>
<tr class="odd">
<td align="center">Enterobacter cloacae</td>
<td align="center">Troleandomycin</td>
</tr>
<tr class="even">
<td align="center">Enterobacter cloacae</td>
<td align="center">Vancomycin</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="interpretation-from-mic-values-disk-diameters-to-rsi" class="section level2">
<h2>Interpretation from MIC values / disk diameters to R/SI</h2>
<p>A data set with 21,996 rows and 10 columns, containing the following column names:<br />
<em>guideline</em>, <em>method</em>, <em>site</em>, <em>mo</em>, <em>ab</em>, <em>ref_tbl</em>, <em>disk_dose</em>, <em>breakpoint_S</em>, <em>breakpoint_R</em> and <em>uti</em>.</p>
<p>This data set is in R available as <code>rsi_translation</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 1 June 2021 14:47:11 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/rsi_translation.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.rds">R file</a> (37 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.xlsx">Excel file</a> (0.7 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.txt">plain text file</a> (1.8 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.sas">SAS file</a> (3.8 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.sav">SPSS file</a> (2.4 MB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/rsi_translation.dta">Stata file</a> (3.5 MB)</li>
</ul>
<div id="source-5" class="section level3">
<h3>Source</h3>
<p>This data set contains interpretation rules for MIC values and disk diffusion diameters. Included guidelines are CLSI (2010-2020) and EUCAST (2011-2021).</p>
</div>
<div id="example-content-5" class="section level3">
<h3>Example content</h3>
<table>
<colgroup>
<col width="8%" />
<col width="5%" />
<col width="4%" />
<col width="12%" />
<col width="20%" />
<col width="17%" />
<col width="7%" />
<col width="9%" />
<col width="9%" />
<col width="4%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">guideline</th>
<th align="center">method</th>
<th align="center">site</th>
<th align="center">mo</th>
<th align="center">ab</th>
<th align="center">ref_tbl</th>
<th align="center">disk_dose</th>
<th align="center">breakpoint_S</th>
<th align="center">breakpoint_R</th>
<th align="center">uti</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">EUCAST 2021</td>
<td align="center">DISK</td>
<td align="center"></td>
<td align="center">Enterobacterales</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Enterobacterales</td>
<td align="center">20-10ug</td>
<td align="center">19</td>
<td align="center">19</td>
<td align="center">FALSE</td>
</tr>
<tr class="even">
<td align="center">EUCAST 2021</td>
<td align="center">DISK</td>
<td align="center">UTI</td>
<td align="center">Enterobacterales</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Enterobacterales</td>
<td align="center">20-10ug</td>
<td align="center">16</td>
<td align="center">16</td>
<td align="center">TRUE</td>
</tr>
<tr class="odd">
<td align="center">EUCAST 2021</td>
<td align="center">MIC</td>
<td align="center"></td>
<td align="center">Enterobacterales</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Enterobacterales</td>
<td align="center"></td>
<td align="center">8</td>
<td align="center">8</td>
<td align="center">FALSE</td>
</tr>
<tr class="even">
<td align="center">EUCAST 2021</td>
<td align="center">MIC</td>
<td align="center">UTI</td>
<td align="center">Enterobacterales</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Enterobacterales</td>
<td align="center"></td>
<td align="center">32</td>
<td align="center">32</td>
<td align="center">TRUE</td>
</tr>
<tr class="odd">
<td align="center">EUCAST 2021</td>
<td align="center">MIC</td>
<td align="center"></td>
<td align="center">Actinomyces</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Anaerobes, Grampositive</td>
<td align="center"></td>
<td align="center">4</td>
<td align="center">8</td>
<td align="center">FALSE</td>
</tr>
<tr class="even">
<td align="center">EUCAST 2021</td>
<td align="center">MIC</td>
<td align="center"></td>
<td align="center">Bacteroides</td>
<td align="center">Amoxicillin/clavulanic acid</td>
<td align="center">Anaerobes, Gramnegative</td>
<td align="center"></td>
<td align="center">4</td>
<td align="center">8</td>
<td align="center">FALSE</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="dosage-guidelines-from-eucast" class="section level2">
<h2>Dosage guidelines from EUCAST</h2>
<p>A data set with 169 rows and 9 columns, containing the following column names:<br />
<em>ab</em>, <em>name</em>, <em>type</em>, <em>dose</em>, <em>dose_times</em>, <em>administration</em>, <em>notes</em>, <em>original_txt</em> and <em>eucast_version</em>.</p>
<p>This data set is in R available as <code>dosage</code>, after you load the <code>AMR</code> package.</p>
<p>It was last updated on 25 January 2021 20:58:20 UTC. Find more info about the structure of this data set <a href="https://msberends.github.io/AMR/reference/dosage.html">here</a>.</p>
<p><strong>Direct download links:</strong></p>
<ul>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.rds">R file</a> (3 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.xlsx">Excel file</a> (14 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.txt">plain text file</a> (15 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.sas">SAS file</a> (52 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.sav">SPSS file</a> (45 kB)<br />
</li>
<li>Download as <a href="https://github.com/msberends/AMR/raw/master/data-raw/../data-raw/dosage.dta">Stata file</a> (44 kB)</li>
</ul>
<div id="source-6" class="section level3">
<h3>Source</h3>
<p>EUCAST breakpoints used in this package are based on the dosages in this data set.</p>
<p>Currently included dosages in the data set are meant for: <a href="https://www.eucast.org/clinical_breakpoints/">‘EUCAST Clinical Breakpoint Tables’ v11.0</a> (2021).</p>
</div>
<div id="example-content-6" class="section level3">
<h3>Example content</h3>
<table>
<thead>
<tr class="header">
<th align="center">ab</th>
<th align="center">name</th>
<th align="center">type</th>
<th align="center">dose</th>
<th align="center">dose_times</th>
<th align="center">administration</th>
<th align="center">notes</th>
<th align="center">original_txt</th>
<th align="center">eucast_version</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">AMK</td>
<td align="center">Amikacin</td>
<td align="center">standard_dosage</td>
<td align="center">25-30 mg/kg</td>
<td align="center">1</td>
<td align="center">iv</td>
<td align="center"></td>
<td align="center">25-30 mg/kg x 1 iv</td>
<td align="center">11</td>
</tr>
<tr class="even">
<td align="center">AMX</td>
<td align="center">Amoxicillin</td>
<td align="center">high_dosage</td>
<td align="center">2 g</td>
<td align="center">6</td>
<td align="center">iv</td>
<td align="center"></td>
<td align="center">2 g x 6 iv</td>
<td align="center">11</td>
</tr>
<tr class="odd">
<td align="center">AMX</td>
<td align="center">Amoxicillin</td>
<td align="center">standard_dosage</td>
<td align="center">1 g</td>
<td align="center">3</td>
<td align="center">iv</td>
<td align="center"></td>
<td align="center">1 g x 3-4 iv</td>
<td align="center">11</td>
</tr>
<tr class="even">
<td align="center">AMX</td>
<td align="center">Amoxicillin</td>
<td align="center">high_dosage</td>
<td align="center">0.75-1 g</td>
<td align="center">3</td>
<td align="center">oral</td>
<td align="center"></td>
<td align="center">0.75-1 g x 3 oral</td>
<td align="center">11</td>
</tr>
<tr class="odd">
<td align="center">AMX</td>
<td align="center">Amoxicillin</td>
<td align="center">standard_dosage</td>
<td align="center">0.5 g</td>
<td align="center">3</td>
<td align="center">oral</td>
<td align="center"></td>
<td align="center">0.5 g x 3 oral</td>
<td align="center">11</td>
</tr>
<tr class="even">
<td align="center">AMX</td>
<td align="center">Amoxicillin</td>
<td align="center">uncomplicated_uti</td>
<td align="center">0.5 g</td>
<td align="center">3</td>
<td align="center">oral</td>
<td align="center"></td>
<td align="center">0.5 g x 3 oral</td>
<td align="center">11</td>
</tr>
</tbody>
</table>
</div>
</div>



<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
